docs/cluster-configuration.md

```diff
@@ -22,6 +22,8 @@ cluster = Cluster(ClusterConfiguration(
     image="quay.io/project-codeflare/ray:latest-py39-cu118", # Mandatory Field
     instascale=False, # Default False
     machine_types=["m5.xlarge", "g4dn.xlarge"],
+    # The labels={} parameter can be used to apply additional labels to the RayCluster resource.
```

Comment:
I think this should be included in the explanatory paragraph underneath rather than here.